openshift/templates/optashift-employee-rostering-template-binary.yaml

```diff
@@ -1,145 +1,114 @@
-    name: employee-rostering
-  spec:
-    replicas: 1
-    selector:
-      app: employee-rostering
-      deploymentconfig: employee-rostering
-    strategy:
-      activeDeadlineSeconds: 21600
-      resources: {}
-      rollingParams:
-        intervalSeconds: 1
-        maxSurge: 25%
-        maxUnavailable: 25%
-        timeoutSeconds: 600
-        updatePeriodSeconds: 1
-      type: Rolling
-    template:
-      metadata:
-        creationTimestamp: null
-        labels:
-          app: employee-rostering
-          deploymentconfig: employee-rostering
-      spec:
-        containers:
-        - image: employee-rostering
-          imagePullPolicy: Always
-          name: employee-rostering
-          ports:
-          - containerPort: 8080
-            protocol: TCP
-          resources: {}
-          terminationMessagePath: /dev/termination-log
-        dnsPolicy: ClusterFirst
-        restartPolicy: Always
-        securityContext: {}
-        terminationGracePeriodSeconds: 30
-    test: false
-    triggers:
-    - type: ConfigChange
-    - imageChangeParams:
-        automatic: true
-        containerNames:
-        - employee-rostering
-        from:
-          kind: ImageStreamTag
-          name: employee-rostering:latest
-      type: ImageChange
-  status:
-    availableReplicas: 0
-    latestVersion: 0
-    observedGeneration: 0
-    replicas: 0
-    unavailableReplicas: 0
-    updatedReplicas: 0
-- apiVersion: v1
-  kind: Service
-  metadata:
-    name: employee-rostering
-  spec:
-    ports:
-    - name: 8080-tcp
-      port: 8080
-      protocol: TCP
-      targetPort: 8080
-    selector:
-      deploymentconfig: employee-rostering
-    sessionAffinity: None
-    type: ClusterIP
-  status:
-    loadBalancer: {}
-- apiVersion: v1
-  kind: Route
-  metadata:
-    name: employee-rostering
-  spec:
-    port:
-      targetPort: 8080
-    to:
-      kind: Service
-      name: employee-rostering
-      weight: 100
-parameters:
-- description: Git source URI for application
-  displayName: Git source repository
-  name: GIT_URI
-  required: true
-  value: https://github.com/kiegroup/optashift-employee-rostering
-- description: Git branch/tag reference
-  displayName: Git branch/tag reference
-  name: GIT_REF
-  required: true
-  value: master
+          name: "webapp-image:latest"
+      resources:
+        limits:
+          cpu: 1
+          memory: 1Gi
+        requests:
+          cpu: 1
+          memory: 1Gi
+
+  - kind: DeploymentConfig
+    apiVersion: v1
+    metadata:
+      name: "webapp-deployment"
+    spec:
+      template:
+        metadata:
+          labels:
+            deploymentconfig: "webapp-deployment"
+        spec:
+          containers:
+          - name: "webapp-container"
+            image: "webapp-image"
+            ports:
+              - containerPort: 8080
+                protocol: TCP
+            resources:
+              limits:
+                cpu: 2
+                memory: 2Gi
```

Comment:
Do we need 2Gi, or did you just set it to the max available?

Reply:
This says we need it. And I find it too unreliable with less - it might work most of the time on 1Gi, but as soon as they start solving multiple tenants - it can go bad.
This does mean we can't deploy this on OpenShift Starter...